x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/util/authz.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SIEM_MIGRATIONS_API_ACTION_ALL } from '@kbn/security-solution-features/actions';
+
+export const authz = {
+  requiredPrivileges: ['securitySolution', SIEM_MIGRATIONS_API_ACTION_ALL],
```

Comment:
Question: I am trying to decide if this requires a privilege migration or new route versions for each of the affected routes.

Users who had roles that previously granted `securitySolution` could perform this functionality, now they need the additional siem privilege.

What do you think? Or am I misunderstanding this?

Reply:
Good point, but this is a new functionality behind a feature flag, it has not been released yet. No need :)

Reply:
Understood. Thanks for clarifying